### DISCUSSION TITLE
PikPak打开报错 {wsarecv: An existing connection was forcibly closed by the remote host.}

### DISCUSSION THREAD

**Discussion by VictoireWood** [AlistGo/alist#1284] [Q&A]
## 前提
- PikPak在[账号管理页面](http://127.0.0.1:5244/@manage/accounts)成功添加；

  <img width="232" alt="成功添加" src="https://user-images.githubusercontent.com/79731222/176138438-24ed1c0c-94b9-4a40-a9bd-ff80e4282c56.png">

- 已挂代理，经测试PikPak可以正常连接；

- 已验证填写PikPak账号的*邮箱地址*和*密码*都准确无误。

## 问题

在[首页](http://127.0.0.1:5244)打开PikPak时报错，如下图所示。

<img width="1072" alt="pikpak打开报错" src="https://user-images.githubusercontent.com/79731222/176137162-be0a787b-114d-4666-9791-6baaa0e17119.png">

>**_错误_**
>Get "https://api-drive.mypikpak.com/drive/v1/files?filters=%7B%22phase%22%3A%7B%22eq%22%3A%22PHASE_TYPE_COMPLETE%22%7D%2C%22trashed%22%3A%7B%22eq%22%3Afalse%7D%7D&limit=100&page_token=&parent_id=&thumbnail_size=SIZE_LARGE&with_audit=true": read tcp 192.168.31.144:4948->47.241.212.52:443: wsarecv: An existing connection was forcibly closed by the remote host.

打开[报错中包含的网址](https://api-drive.mypikpak.com/drive/v1/files?filters=%7B%22phase%22%3A%7B%22eq%22%3A%22PHASE_TYPE_COMPLETE%22%7D%2C%22trashed%22%3A%7B%22eq%22%3Afalse%7D%7D&limit=100&page_token=&parent_id=&thumbnail_size=SIZE_LARGE&with_audit=true)后，显示的信息如下。

<img width="747" alt="pikpakAPI报错" src="https://user-images.githubusercontent.com/79731222/176137153-d6f0b29f-cadc-47e0-b460-4561b3da2edc.png">

```json
{"error":"unauthenticated","error_code":16,"error_url":"","error_description":"帐号认证失败，请重新登录","error_details":[{"@type":"type.googleapis.com/google.rpc.DebugInfo","stack_entries":[],"detail":"invalid auth header"}]}
```

请问大佬们这种情况应该怎么解决？十分感谢！🙏🏻

**Comment by xhofe** [comment]
https://www.google.com/search?q=how+to+set+proxy+in+terminal

**Reply by VictoireWood** [reply]
那Windows里想要代理设置只对`alist-windows-4.0-amd64.exe`生效的话，我应该把设置文件放在哪里呢？
我是需要让TCP对`192.168.31.144:4948->47.241.212.52:443`建立通信吗？
